.github/workflows/update-moon-landing.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "js/**"
```

Comment:
i had understood it was going to be triggered manually?

otherwise it's gonna be spammy in 🌔  no?

Reply:
The action creates a PR and maintains it up to date. So we'll have only one PR which will be updated continuously until it's merged

Reply:
ah ok

that's cool

Maybe document this for posterity

Reply:
you should remove the "js/**" path because we are also using tasks folder, and even docs (i'm parsing the licenses table  at runtime 🤯 )